src/index.js

```diff
@@ -0,0 +1,32 @@
+import './style.css';
+
+function component() {
+  const element = document.createElement('div');
+
+  const eachTodoList = [
+    {
+      description: 'cooking',
+    },
+    {
+      description: 'Cleaning the house',
+    },
+    {
+      description: 'Going to gym',
+    },
+  ];
```

Comment:
- [ ] Good job setting up webpack, please see as mentioned in the project requirement we need to create an array of objects where each object will have three keys, description of type [string], completed of type [boolean], index of type [number]
![image](https://user-images.githubusercontent.com/97521557/222215582-4259b983-14c6-4a1a-9ada-66e3651f70d4.png)
 example : 
```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```